### ISSUE TITLE
image templates need to clean up /dev/sensors

### ISSUE THREAD

**Issue by rmustacc** [oxidecomputer/helios#159]
While working on cleaning up the smntemp related issues, I was surprised to see a Gimlet that had booted with the following entries in `/dev/sensors`:

```
BRM42220071 # find /dev/sensors/
/dev/sensors/
/dev/sensors/temperature
/dev/sensors/temperature/cpu
/dev/sensors/temperature/cpu/procnode.0
/dev/sensors/temperature/pci
/dev/sensors/temperature/pci/c3.0
/dev/sensors/temperature/pci/c3.0/builtin
```

It took me a bit; however, I found that the sensor link devfsadm module was not present at all nor was the ksensor driver! This took me on a bit of a hunt, but the image-builder is running devfsadm and has a comment that we need to clean up unused directories. So we should add something to remove this directory. Separately, we should probably ship ksensor and smntemp, but that's a different day's bug!

**Comment by rmustacc** [comment]
It's worth noting that neither of these corresponded to anything in the kernel's `ksensor_avl`. When I booted up a system with a fix for this I now see:

```
BRM42220071 # ls -l /dev/sensors
/dev/sensors: No such file or directory
```

This makes sense and corresponds to what I think we expect given we don't have the devfsadm plugin present. I then manually installed the ksensor driver, .conf file, and linkmod and saw expected sensors:

```
BRM42220071 # find /dev/sensors/
/dev/sensors/
/dev/sensors/temperature
/dev/sensors/temperature/pci
/dev/sensors/temperature/pci/85.0
/dev/sensors/temperature/pci/85.0/temp
/dev/sensors/voltage
/dev/sensors/voltage/pci
/dev/sensors/voltage/pci/85.0
/dev/sensors/voltage/pci/85.0/vdd
```

**Comment by jclulow** [comment]
Whoops!  I do wonder more generally if we shouldn't have some way to prime /dev without engaging any plugins that detect actual hardware on the system, for the image construction use case.  Potentially even just that all of the static stuff that absolutely must exist prior to boot just gets packaged and then we don't have to run the tool at all this way.

**Comment by rmustacc** [comment]
I suspect that having that for image construction would be useful longer term. In the interim, I'd like to land this ahead of adding the ksensor related stuff for real.

**Comment by jclulow** [comment]
Yes definitely

**Comment by rmustacc** [comment]
With the final changes requested, there is now an empty `/dev/sensors` directory:

```
rm@cadbury ~ $ ssh root@c71
Warning: Permanently added 'fe80::eaea:6aff:fe09:7fab%e1000g0' (ED25519) to the list of known hosts.

    #####
   ##   ##
  ##   # ##  ##   ##
  ##  #  ##   ## ##     Oxide Computer Company
  ## #   ##    ###      Engineering
   ##   ##    ## ##    
    #####    ##   ##    Gimlet

BRM42220071 # ls -l /dev/sensors
total 0
```

**Comment by rmustacc** [comment]
Addressed via #159.
